reporter/src/main/kotlin/reporters/ReportTableModelMapper.kt

```diff
@@ -103,11 +117,8 @@ class ReportTableModelMapper(private val resolutionProvider: ResolutionProvider)
             val tableRows = allIds.map { id ->
                 val scanResult = scanRecord?.scanResults?.find { it.id == id }
 
-                val scopes = project
-                    .scopes
-                    .filter { id in it }
-                    .map { it.name }
-                    .let { scopesNames -> excludes.scopeExcludesByName(scopesNames).toSortedMap()
+                val scopes = scopesForDependencies[id]!!.let { scopesNames ->
```

Comment:
Consider using `getValue` instead of `!!`.

Reply:
Done.